index.html

```diff
@@ -0,0 +1,184 @@
+                    </div>
+                </div>
+                <div class="box">
+                    <img width="70px" height="100px" src=" images\fun-logo-vector-smile-emotion-260nw-344718320.webp"
+                        alt="">
+                    <h4 class="box-main">Fun</h4>
+                    <div class="box-text">
+                        It sure'd be fun with lots of musicians come together and perform
+                    </div>
+                </div>
+                <div class="box">
+                    <img width="70px" height="100px"
+                        src="images\money-logo-design-vector-illustrative-260nw-1810454230.webp" alt="">
+                    <h4 class="box-main">Free</h4>
+                    <div class="box-text">
+                        Pay no price whatsoever your punctuality is all that matters about getting in.
+                    </div>
+                </div>
+            </div>
+        </section>
+        <section class="featured-speaker-container">
+            <h1 class="featuring">Featuring</h1>
+
+            <div class="feature-box-container">
+                <div class="feature-box">
+                    <img class="image cimage" src="" alt="">
+                    <div class="featur-box-content">
+                        <div class="feature-box-text cname"></div>
+                        <div class="feature-box-text1 cdescription"></div>
+                        <div class="feature-box-text2 coverview">
+                        </div>
+                    </div>
+
+                </div>
+                <div class="feature-box">
+                    <img class="image rimage" src="" alt="">
+                    <div class="feauture-box-content">
+                        <div class="feature-box-text rname"></div>
+                        <div class="feature-box-text1 rdescription"></div>
+                        <div class="feature-box-text2 roverview">.</div>
+                    </div>
+
+                </div>
+                <div class="feature-box">
+                    <img class="image simage" src="" alt="">
+                    <div class="feature-box-content">
+                        <div class="feature-box-text sname"></div>
+                        <div class="feature-box-text1 sdescription"></div>
+                        <div class="feature-box-text2 soverview">
+                        </div>
+                    </div>
+
+                </div>
+                <div class="feature-box last">
+                    <img class="image stimage" src="" alt="">
+                    <div class="feature-box-content">
+                        <div class="feature-box-text stname"></div>
+                        <div class="feature-box-text1 stdescription"></div>
+                        <div class="feature-box-text2 stoverview">
+                        </div>
+                    </div>
+                </div>
+            </div>
+
+        </section>
```

Comment:
- [ ] this section should be generated dynamically with javascript please try to remove all the elements in this section and try to generate anything in this section with javascript